code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -108,22 +127,23 @@
 		if(pressure_delta > 0)
 			if(air_contents.temperature > 0)
 				var/transfer_moles = pressure_delta*environment.volume/(air_contents.temperature * R_IDEAL_GAS_EQUATION)
-
+				transfer_moles = transfer_moles * transfer_modifier
```

Comment:
What is the intent here? And why in the name of god are you multiplying?

Reply:
like i said, quick test, i thought that increasing the transfer_moles amount would increase the amount of gas expelled from the vent

Reply:
yep, gottcha